jenkins/kola/aws.sh

```diff
@@ -44,6 +44,33 @@ if [[ "${AWS_AMI_ID}" == "" ]]; then
   echo "Created new AMI ${AWS_AMI_ID} (will be removed after testing)"
 fi
 
+# Run the cl.internet test on multiple machine types only if it should run in general
+cl_internet_included="$(set -o noglob; bin/kola list --platform=aws --filter ${KOLA_TESTS} | { grep cl.internet || true ; } )"
+if [[ "${BOARD}" == "amd64-usr" ]] && [[ "${cl_internet_included}" != ""  ]]; then
+  for INSTANCE in m4.2xlarge; do
+    (
+    set +x
+    OUTPUT=$(timeout --signal=SIGQUIT 6h bin/kola run \
+    --parallel=8 \
+    --basename="${NAME}" \
+    --board="${BOARD}" \
+    --aws-ami="${AWS_AMI_ID}" \
+    --aws-region="${AWS_REGION}" \
+    --aws-type="${INSTANCE}" \
+    --aws-iam-profile="${AWS_IAM_PROFILE}" \
+    --platform=aws \
+    --channel="${GROUP}" \
+    --offering="${OFFER}" \
+    --tapfile="${JOB_NAME##*/}_validate_${INSTANCE}.tap" \
+    --torcx-manifest=torcx_manifest.json \
+    cl.internet 2>&1 || true)
+    echo "=== START $INSTANCE ==="
+    echo "${OUTPUT}" | sed "s/^/${INSTANCE}: /g"
+    echo "=== END $INSTANCE ==="
```

Comment:
Would it be less likely to have a mixed output if we did the output with one command? Feel free to ignore it if it does not matter.

```suggestion
    printf '=== START %s ===\n%s\n=== END %s ===\n' "$INSTANCE" "$(echo "${OUTPUT}" | sed "s/^/${INSTANCE}: /g")" "$INSTANCE"
```

Reply:
Yes, a bit less likely but I don't think we hit the case often where two runs exit on exactly the same time. But even then I'm not sure that it helps because two concurrent printfs will also fight with each other, or?

Reply:
I'd expect the submission to some buffer to be done under some lock or whatever, so there would be a chance that printf would do such a submission once. But who knows, let's get this in. If we have an issue with a mangled output, then we will worry about it.